tools/pub_get_offline.py

```diff
@@ -64,7 +65,19 @@ def fetch_package(pub, package):
   return 0
 
 
-def check_package(package):
+def package_uses_workspace_resolution(package):
+  pubspec = os.path.join(package, 'pubspec.yaml')
```

Comment:
I see [pyyaml in the DEPS](https://github.com/flutter/engine/blob/08058e331a43477a06e4a5613566c41765712803/DEPS#L669). Is that not usable?

Reply:
Sounds good, I used the same pattern as `gen_dart_package_config.py` and went with:

```py
#!/usr/bin/env vpython3

# [VPYTHON:BEGIN]
# python_version: "3.8"
# wheel <
#   name: "infra/python/wheels/pyyaml/${platform}_${py_python}_${py_abi}"
#   version: "version:5.4.1.chromium.1"
# >
# [VPYTHON:END]
```

Reply:
We should only be fetching dependencies from the network from the DEPS file.

Reply:
Right, pubspec files with `resolution: workspace` do not fetch dependencies at all.

This script will throw saying "no package_config.json", hence this change.

Reply:
Ah, sorry. My comment was in reference to the discussion of pyyaml and using the wheel.